src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
+
+    &:hover {
+      color: #00acdc;
+    }
```

Comment:
maybe would be nice for that hover to work when I point with the mouse under or over the link with the hover will work.